wiremock-docker/__files/basm-api/locations/COURT2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "data": [
+    {
+      "id": "7bad3789-0d49-44dc-bb8f-4ff32414fa0b",
```

Comment:
Will non-unique IDs cause issues?

Reply:
Nope as we won't use them, we only care about the location name.  I'll change them though anyway.